static/src/javascripts/lib/geolocation.js

```diff
@@ -4,7 +4,6 @@ import config from 'lib/config';
 import { local as storage } from 'lib/storage';
 
 const storageKey = 'gu.geolocation';
-export type geolocationString = 'GB' | 'US' | 'AU';
```

Comment:
Removing enum type as it is not all geolocation strings but limited to those relevant for reader revenue and so confusing. This type was only used in one test.